ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionIndexing.scala

```diff
@@ -209,6 +209,12 @@ object TransactionIndexing {
       val allContractIds = allCreatedContractIds.union(archived)
       val netCreates = created.filterNot(c => archived(c.coid))
       val netArchives = archived.filterNot(allCreatedContractIds)
+      val netKeyNullifies = netCreates.flatMap(create =>
```

Comment:
This seems slightly off. I think it makes more sense to serialize later like we do for everything else afaict.

Reply:
Done